charts/pravega-operator/README.md

```diff
@@ -44,6 +45,8 @@ $ helm uninstall [RELEASE_NAME]
 
 This command removes all the Kubernetes components associated with the chart and deletes the release.
 
+>Note: If you want to delete the Pravega clusters, make sure to do it before uninstalling the operator.
+
 ## Configuration
```

Comment:
Please call this section "Operator-Configuration"

Reply:
Done